samples/web/content/apprtc/apprtc.py

```diff
@@ -342,6 +348,8 @@ def add_client_to_room(host, room_id, client_id, is_loopback):
     if memcache_client.cas(key, room, constants.ROOM_MEMCACHE_EXPIRATION_SEC):
       logging.info('Added client %s in room %s, retries = %d' \
           %(client_id, room_id, retries))
+      if room.get_occupancy() == 2:
+        analytics.report_impression('room_size_2', room_id, host=request.host)
```

Comment:
This string should also go to a const with documentation on what this impression means.